galasa-parent/dev.galasa.framework.api.ras/openapi.yaml

```diff
@@ -263,4 +288,11 @@ components:
     JsonError:
       properties:
         error:
-          type: string
\ No newline at end of file
+          type: string
+    TestClasses:
+      type: object
+      properties:
```

Comment:
this needs to be an array of objects, which contains testclass:"xxxx",  bundle:"yyyyy"
So the end result looks like:-
{testclasses:[{testclass:"xxx",bundle:"yyyy"},{testclass:"xxx",bundle:"yyyy"}]}